drivers/pure/flasharray/pods.go

```diff
@@ -0,0 +1,28 @@
+package flasharray
+
+type PodServices struct {
+	client *Client
+}
+
+func (p *PodServices) CreatePod(params map[string]string, data interface{}) (*[]PodResponse, error) {
```

Comment:
If we're going to create Pods, make sure we also delete them - Pods are a fairly limited resource on the array, we can't afford to let them stack up for all jobs, only failed ones

Reply:
Added it